exec/java-exec/src/main/resources/options.describe.properties

```diff
@@ -0,0 +1,107 @@
+#

```

Comment:
Since we have now options.describe.properties file, how did we agree to enforce developers to add descriptions?
Will project build fail if description was not added? Or maybe instead of using properties file, we can make `OptionValidator` to accept description along with option name, thus all options will be required to have description? And we won't have to duplicate options names in options.describe.properties?

Reply:
For future additions, we could do this, but I didn't want to right away change the existing OptionValidator usages. I'll need to work with @bbevens to fill up the properties file for now, since total list of options are 150+.

Reply:
We can log a new JIRA for updating the Options validator so that all options have a description and move on.

Reply:
Sorry guys but I think it's better not to introduce `options.describe.properties` file at all. Updating Options validator is not that hard, for those options we don't have a description prepared, just leave blanks.

Reply:
@arina-ielchiieva the purpose of the PR was a UX enhancement. The backend is a temporary change until the proposed change to the OptionValidator is implemented later. Time to change existing OptionValidators would require significant time. Introducing the complete change now, by leaving blanks defeats the purpose of the UX enhancement.

Reply:
@kkhatua usually there is nothing more persistent then temporary :)
Current list of options in properties files does not contain full list of system / session options. It means that the rest would have empty description, right? So I am not sure how this is better then updating OptionsValidators. Expanding the constructor and adding descriptions to options, does not seem to take a significant time, might be not the most interesting part of the development but still.

Reply:
Yes, some options have empty descriptions because I didn't find them on the website. I'm looking to get in the UX enhancement originally intended for 1.14.0. I also wanted to (separately) discuss whether changes to the contructor is suitable or introduction of annotations.

Reply:
I would go with constructor.